### PR TITLE
Stracec

### PR DESCRIPTION
**Pull request by rupeshkoushik07**
## Description

Fixes # ([issue](https://github.com/Lind-Project/lind_project/issues/345))

- Adds the stracec_increment function for improved code quality.
- Replaces elapsedTime with totalTime to ensure uniformity

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to a pull request and/or merged in other modules (lind-project, lind-glibc, safeposix-rust)

### CODE REVIEW

File: src/trusted/service_runtime/nacl_syscall_strace.c [hunk 1]
```diff
@@ -96,143 +96,108 @@ char * formatStringArgument(const char * input) {
...
+        syscallStats[syscallnum].errorCount++;
+    }
+}
+
+void NaClStraceGetpid(int cageid, int pid, long long totaltime) {
   if (strace_C) {
-    syscallStats[NACL_sys_getpid].count++;
-    syscallStats[NACL_sys_getpid].totalTime += elapsedTime;
+    stracec_increment(NACL_sys_getpid, totaltime, 0);
     }else {
```
Comment (rennergade) [inline src/trusted/service_runtime/nacl_syscall_strace.c]: some of these } else statements need a space between the bracket

**Review by rennergade — CHANGES_REQUESTED** [review]
looks really good! just need a minor cosmetic fix

File: src/trusted/service_runtime/nacl_syscall_strace.c (hunk 1 above)
Comment (rupeshkoushik07) [reply src/trusted/service_runtime/nacl_syscall_strace.c]: I have updated the spacing

**Review by rennergade — APPROVED** [review]
looks great!

**Review by rennergade — APPROVED** [review]
Looks great. Thanks rupesh!

**Pull request merged**